tox.ini

```diff
@@ -90,6 +97,12 @@ deps =
     Django<1.9
     {[testenv]deps}
 
+[testenv:py33-django19]
```

Comment:
this environment can be remove since py33 support is dropped in 1.9

https://docs.djangoproject.com/en/1.8/faq/install/#what-python-version-can-i-use-with-django